### PR TITLE
Nam integrating

### PR DESCRIPTION
**Pull request by namvuCosmo**
added classes to with with kite server and a DB. tested with engine 2.0.3, framework 1.0.1, with the usual tests.

### CODE REVIEW

File: src/main/java/io/cosmosoftware/kite/dao/DaoManager.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+/**
+ * The Class DaoManager.
+ *
+ * @param <T>
+ *            the generic type
+ */
+public class DaoManager<T> {
+
+	/** The Constant logger. */
```
Comment (ManuCosmo) [inline src/main/java/io/cosmosoftware/kite/dao/DaoManager.java]: code style

File: pom.xml [hunk 2]
```diff
@@ -165,6 +169,29 @@
       <artifactId>monte-repack</artifactId>
       <version>${kite.pojosontheweb.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.hibernate.javax.persistence</groupId>
```
Comment (ManuCosmo) [inline pom.xml]: can we avoid adding these dependencies to kite-extras if they are only needed for the kiteServer and not the rest?

File: src/main/java/io/cosmosoftware/kite/interfaces/CommandMaker.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2018 Cosmo Software
```
Comment (ManuCosmo) [inline src/main/java/io/cosmosoftware/kite/interfaces/CommandMaker.java]: wrong copyright

File: pom.xml (hunk 2 above)
Comment (namvuCosmo) [reply pom.xml]: I wouldn't put it if it could be avoided

File: src/main/java/io/cosmosoftware/kite/interfaces/CommandMaker.java (hunk 3 above)
Comment (namvuCosmo) [reply src/main/java/io/cosmosoftware/kite/interfaces/CommandMaker.java]: 2019 ? I copied the file from kite server

Comment (ManuCosmo) [reply src/main/java/io/cosmosoftware/kite/interfaces/CommandMaker.java]:
```
/*
 * Copyright (C) CoSMo Software Consulting Pte. Ltd. - All Rights Reserved
 */

```

**Pull request merged**